packages/vscode-extension/package.json

```diff
@@ -1106,7 +1107,7 @@
     "copy-files": "copyfiles -u 1 src/**/*.html src/**/*.css out/src/",
     "copy-whatisnew-files": "copyfiles WHATISNEW.md out/resource",
     "compile": "tsc -p ./ && npm run copy-files && npm run copy-whatisnew-files",
-    "build": "rimraf out && webpack --mode development --config ./webpack.config.js && npm run compile",
+    "build": "tsc -p ./",
```

Comment:
Changing the build process to tsc will not work for webviews.

Reply:
Thank you! This is not intended change, will fix it.